docs/sources/enterprise/request-security.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Request security"
+description = "Grafana Enterprise request security"
+keywords = ["grafana", "security", "enterprise"]
+weight = 110
++++
+
+# Request security
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+Request security introduces ways to limit requests from the Grafana server, and it targets requests that are generated by users.
+
+Example:
+- data source metric queries
+- alert notifications
+
+This can be used to limit access to internal systems that the server Grafana runs on can access but that users of Grafana should not be able to access. This feature does not affect traffic from the Grafana users browser.
+
+Note: Although request security works with backend plugins, it is possible to create a backend plugin that bypasses this security.
+
+## IP and hostname blocking
+
+It is possible to limit requests based on a hostname, an IP address, or both.
+
+### Deny list
+
+Any request to a hostname or IP address on the deny-list is denied.
+
+### Allow list
+
+The allow list is only activated if there is at least one hostname or IP address on it. Furthermore, any request to a hostname or IP address that is not on the allow list is denied.
```

Comment:
```suggestion
The allow list is activated when there is at least one hostname or IP address on it. Furthermore, any request to a hostname or IP address that is not on the allow list is denied.
```

Reply:
I reserve "when" to handle time-based contexts. For that reason, -1.

Reply:
Per @oddlittlebird , if should always have a "then". So, that is what I have been following. Let's discuss these in detail. Otherwise, we are all following different review styles based on what we like?

Reply:
Here, the construction is "only if" rather than "if then".

Reply:
Correct. I'm also not a fan of "furthermore," but the if/then pairing is something I stick with.

Out of curiosity, how does one activate a list? Perhaps we are talking about the wrong thing here. Would it be more correct to say, "If there is at least one entry on the list, then any request to a hostname or IP address not on the list is denied"?

Reply:
@xlson from a minimal viable improvement perspective, pick one and we can go over this offline.

Reply:
@oddlittlebird I like that version, reads a bit better to me.